### PR TITLE
add list and activity type

### PR DESCRIPTION
**Pull request by kazuki-yane**
* Add list and activity_type to target

### CODE REVIEW

**Comment by kekekenta** [comment]
Would you modify README.md

File: src/test/java/org/embulk/input/marketo/delegate/ActivityTypeInputPluginTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
```
Comment (kekekenta) [inline src/test/java/org/embulk/input/marketo/delegate/ActivityTypeInputPluginTest.java]: unnecessary comment

File: src/test/java/org/embulk/input/marketo/delegate/ListInputPluginTest.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ * Created by tai.khuu on 10/10/17.
```
Comment (kekekenta) [inline src/test/java/org/embulk/input/marketo/delegate/ListInputPluginTest.java]: unnecessary comment

**Review by kekekenta — APPROVED** [review]

**Comment by giwa** [comment]
enabled travisCI. It shows green light:)

**Review by giwa — APPROVED** [review]

**Pull request merged**